flake.nix

```diff
@@ -20,10 +20,10 @@
           name = "http4s";
           typelevelShell = {
             jdk.package = jdk;
-            nodejs.enable = false;
+            nodejs.enable = true;
             native.enable = true;
+            native.libraries = [ pkgs.zlib pkgs.s2n-tls pkgs.openssl ];
```

Comment:
Is there a way to append here?

Reply:
I don't think so.  The module system supports merges of imported modules, and that concatenates lists.  But `[ pkgs.zlib ]` is just a default if nothing specifies that key.